actiontext/test/template/form_helper_test.rb

```diff
@@ -25,6 +25,21 @@ def form_with(*, **)
     )
   end
 
+  test "rich text area tag" do
+    message = Message.new
+    form_with model: message, scope: :message do |form|
```

Comment:
We generally prefer to leave blank lines around multiline method calls:

```suggestion
    message = Message.new

    form_with model: message, scope: :message do |form|
```